actiontext/lib/action_text/content.rb

```diff
@@ -88,7 +92,7 @@ def to_html
     end
 
     def to_rendered_html_with_layout
-      renderer.render(partial: "action_text/content/layout", locals: { content: self })
+      renderer.render(layout: "action_text/content", partial: self)
```

Comment:
@georgeclaghorn this felt like a good compromise: make the `app/views/layouts/action_text/_content.html.erb` template part of the "public API", but reduce the surface area of its interface down to "where should I put the `yield` call?" while keeping the original call to `render_action_text_content(content)` as an internal implementation detail.

Is there prior art for exposing a `layout/$MODULE_NAME` template to user-land customizations?